src/components/homepage/Homepage.js

```diff
@@ -23,6 +24,7 @@ export class Homepage extends Component {
           <Grid.Row style={{ height: '320px' }}>
           </Grid.Row>
         </Grid>
+        <Footer />
```

Comment:
I think we should be rendering this on `App.js`?
I think we want the footer to appear on all pages

Reply:
@mattwr18 Ah, you're right.

Let me fix it.

Reply:
@mattwr18 Done!

Sorry for the inconvenience, can you PTAL?

Reply:
Awesome! thanks!! :smile_cat: